dashboard/lib/tasks/seed.rake

```diff
@@ -64,7 +64,7 @@ namespace :seed do
 
   # detect changes to dsldefined level files
   # LevelGroup must be last here so that LevelGroups are seeded after all levels that they can contain
-  DSL_TYPES = %w(TextMatch ContractMatch External Match Multi LevelGroup)
+  DSL_TYPES = %w(TextMatch ContractMatch External Match Multi EvaluationMulti LevelGroup)
```

Comment:
The best way would be if there is some way to get the entire class structure, and make this everything descended from DSLDefined